ui/cypress/tests/datalake/widgets/2d-correlation.spec.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { DataLakeUtils } from '../../../support/utils/datalake/DataLakeUtils';
+
+describe('Test 2d Correlation View in Data Explorer', () => {
+    beforeEach('Setup Test', () => {
+        cy.initStreamPipesTest();
+        DataLakeUtils.loadRandomDataSetIntoDataLake();
+    });
+
+    it('Perform Test', () => {
+        DataLakeUtils.goToDatalake();
+
+        DataLakeUtils.createAndEditDataView('view');
+
+        DataLakeUtils.addNewWidget();
+
+        DataLakeUtils.selectDataSet('Persist');
+
+        DataLakeUtils.dataConfigSelectAllFields();
+
+        DataLakeUtils.selectVisualizationConfig();
+
+        DataLakeUtils.selectVisualizationType('2D Correlation');
+
+        DataLakeUtils.clickCreateButton();
+
+        DataLakeUtils.selectTimeRange(
```

Comment:
Is it possible to add this also to the new utils function? Maybe it is possible to change the time range before adding the widget.